calibration_publisher_node:
  ros__parameters:
    target_frame: World
    child_frame: camera
    translation:
    - 0.22322965882965448
    - -3.5172613475558117
    - 2.7403745695388997
    rotation_matrix:
    - - 0.9979215186564693
      - -0.06273246637208667
      - -0.014740429615232177
    - - -0.03566932497935461
      - -0.7282291228350057
      - 0.6844048830264695
    - - -0.053668816438261616
      - -0.6824565790713965
      - -0.7289531369191597
